### DISCUSSION TITLE
Differentiate manual payments by type.

### DISCUSSION THREAD

**Discussion by rickcj** [pretix/pretix#4040] [Q&A]
Is it possible to specify the payment method used for manual payment (cash, card) for customers purchasing in person?

**Comment by pc-coholic** [comment]
No, using the pretix backend to mark an order as paid is just a "last resort" means.

For proper "pay at venue", you should use [pretixPOS](https://pretix.eu/about/en/pos) which does indeed record the the payment method details (cash, card - and which card payment provider).

**Reply by rickcj** [reply]
We are a small charity community centre and don't have the budget for that and some of the staff are in the 80's and so using PretixPOS would be too complicated.

**Reply by pc-coholic** [reply]
I'm sorry - there isn't any other way at this point...

Since pretix has a very well documented API, you can of course tack on your own interface to create payments that include more information...

Not trying to sell you anything - but did you try out pretixPOS? I might be biased as one of the developers - but I feel like POS is *a lot* easier to use than the web shop or backend. And a lot faster, too... I the simplest of cases, a checkout is three taps: select product, tap on "checkout", tap on payment method...

And to conclude this comment with some advertising: pretix.eu does also offer customized pricing if the default one does not work for you ;-)
